hack/boilerplate/boilerplate_test.py

```diff
@@ -16,7 +16,7 @@
 
 import boilerplate
 import unittest
-import StringIO
+from io import StringIO
```

Comment:
seems unrelated? how did the go bump switch this to python3?

Reply:
Bazel was failing this test locally on my side when I bumped things. May be my setup?

Reply:
ah. I thought we had it configured to force python2 /shrug
as long as it passes in CI, seems good to me